Jenkinsfile.build

```diff
@@ -88,7 +93,9 @@ pipeline {
                             """
                         }                    
                     }
-                }
+                 }
```

Comment:
Could you tighten up the whitespace here?

Reply:
I cleaned things up a bit